modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/SelectStoreQueryRuntime.java

```diff
@@ -52,7 +52,8 @@ public SelectStoreQueryRuntime(QueryableProcessor queryableProcessor, CompiledCo
     public Event[] execute() {
         try {
             StateEvent stateEvent = new StateEvent(1, 0);
-            StreamEvent streamEvents = queryableProcessor.query(stateEvent, compiledCondition, compiledSelection);
+            StreamEvent streamEvents = queryableProcessor.query(stateEvent, compiledCondition,
+                    compiledSelection, outputAttributes);
```

Comment:
Cant we move the outputAttributes into the compiledSelection, so we don't need to change the APIs?